### PR TITLE
Update deb package version

### PR DESCRIPTION
**Pull request by hsbasu**
- Read version from deb packaging for meson build

### CODE REVIEW

**Review by amit9838 — APPROVED** [review]

File: meson.build [hunk 1]
```diff
@@ -1,5 +1,5 @@
 project('mousam',
-          version: '1.0.1',
+          version: run_command('dpkg-parsechangelog',  '--show-field', 'Version', check: true).stdout(),
```
Comment (amit9838) [inline meson.build:2]: @hsbasu can you pls check this again, I merged it in the master, flatpak build was unsuccessfull for some reason

Comment (amit9838) [reply meson.build:2]: Actually Dynamic versioning is breaking the flatpak build, and snap too,  so we have reverted it to original convention. can you check this again

**Review by amit9838 — COMMENTED** [review]
added comments related to some issue

**Comment by hsbasu** [comment]
Yeah. I also found this after you have merged. I prepared the fix. I'll create a PR. Also, while building flatpak `dpkg-dev` would be needed as a build dependency.

File: meson.build (hunk 1 above)
Comment (hsbasu) [reply meson.build:2]: while building flatpak `dpkg-dev` would be needed as a build dependency.

Comment (amit9838) [reply meson.build:2]: Is there any way, to get version information in deb build without modifying the meson.build file?

Comment (hsbasu) [reply meson.build:2]: I have created a local script to parse the version rather than using `dpkg-parsechangelog`.

Comment (hsbasu) [reply meson.build:2]: I'll create a PR today with these fixes.

**Pull request merged**